CMakeLists.txt

```diff
@@ -142,7 +142,7 @@ endif()
 set( MIOPEN_BACKEND ${MIOPEN_DEFAULT_BACKEND} CACHE STRING
     "Which of MIOpens's backends to use?" )
 set_property( CACHE MIOPEN_BACKEND PROPERTY STRINGS
-    OpenCL HIP HIPOC )
+    OpenCL HIP HIPOC NOGPU)
```

Comment:
So maybe this should be name `HIPNOGPU` since it still requires hip runtime but no gpu.

Reply:
Fixed